pkg/config/validation/validation.go

```diff
@@ -1994,6 +1994,13 @@ func validateHTTPStatus(status int32) error {
 	return nil
 }
 
+func validateGPRCStatus(status string) error {
+	if _, ok := grpcutil.SupportedGRPCStatus[status]; !ok {
```

Comment:
I suspect a user may get confused by this. Can we expand this to say something like:

`gRPC status "foo" is not supported`. See https://github.com/grpc/grpc/blob/master/doc/statuscodes.md for a list of supported codes, for example "NOT_FOUND"`

This makes it clear that `not_found` or `5` are not valid.

Reply:
Can you add this?